### PR TITLE
Add Pocket Alert

### PR DESCRIPTION
**Pull request by Akellacom**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -783,6 +783,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [webpushr](https://www.webpushr.com/) - Web Push Notifications - Free for upto 10k subscribers, unlimited push notifications, in-browser messaging
   * [httpSMS](https://httpsms.com) - Send and receive text messages using your Android phone as an SMS Gateway. Free to send and receive up to 200 messages per month.
   * [EMQX Serverless](https://www.emqx.com/en/cloud/serverless-mqtt) - Scalable and secure serverless MQTT broker you can get in seconds. 1M session minutes/month free forever (no credit card required).
+  * [Pocket Alert](https://pocketalert.app) - Send push notifications to your iOS and Android devices. Effortlessly integrate via API or Webhooks and maintain full control over your alerts. Free plan: 50 messages per day.
```
Comment (ripienaar) [inline README.md:786]: please add to the limits "50 messages per day to 1 device and 1 application"

Comment (Akellacom) [reply README.md]: @ripienaar I apologize, I forgot to include the full details of the free plan. i've updated!

**Comment by ripienaar** [comment]
Thanks

**Review by Shahid439-max — COMMENTED** [review]
README.md

**Pull request merged**